### PR TITLE
Fix typo in entry point name

### PR DESCRIPTION
**Pull request by timj**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by timj** [comment]
I'm not sure why #7 reverted to an old version although 21.3.0 is the version I have installed.

**Comment by ericpre** [comment]
@conda-forge-admin, please rerender

**Comment by timj** [comment]
Should I re-add the current version of towncrier to this?

**Comment by ericpre** [comment]
Up to you, this can be done here or in a separate PR.

**Comment by timj** [comment]
Okay. I'm not sure what is causing this:

```
ModuleNotFoundError: No module named 'pkg_resources'
```
in the check.

**Comment by timj** [comment]
Turns out that for py310 we need an explicit setuptools dependency. That fixed it.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {% set name = "towncrier" %}
-{% set version = "19.9.0" %}
-{% set sha256 = "19916889879353a8863f3de8cb1ef19b305a0b5cfd9d36159d76ca2fef08e9aa" %}
+{% set version = "21.3.0" %}
+{% set build_number = 1 %}
```
Comment (ericpre) [inline recipe/meta.yaml]: Better to remove this line and edit directly the build number line 16. At the moment, the variable `build_number` is not being used and therefore the build number is still 0 as defined line 16 - you can check in the build log on azure pipelines.

Comment (timj) [reply recipe/meta.yaml]: Moved. sorry about that.

**Review by ericpre — APPROVED** [review]

**Pull request merged**